src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -152,7 +154,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
             {
                 IWcfService proxy = factory.CreateChannel();
                 watch.Start();
-                proxy.EchoWithTimeout("Hello");
+                proxy.EchoWithTimeout("Hello", serviceOperationTimeout);
             });
         }
         finally
```

Comment:
If we feel that timing that out at 4992 ms is actually an HttpClient bug in NET Native, I recommend we open another bug for it.  Possible or no, I see this failure on N about once a week.  I doubt we can make a reliable repro.  It is conceivable some rounding error in the way they setup their timeout that might give different behavior depending exactly when the timer starts.  I've never seen it more than about 10ms too short.